<launch>
  <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
  <param name="serial_port"         type="string" value="/dev/ttyUSB0"/>
  <param name="serial_baudrate"     type="int"    value="115200"/><!--A1/A2 -->
  <param name="frame_id"            type="string" value="laser"/>
  <param name="inverted"            type="bool"   value="false"/>
  <param name="angle_compensate"    type="bool"   value="true"/>
  </node>
	<include file="$(find laser_filters)/launch/median_filter.launch" />
	<node name="segmentation_laserscan"          pkg="segmentation_laserscan"  type="segmentation_node.py" output="screen"/>
	<node name="classification_segments"          pkg="classification_segments"  type="classification_node.py" output="screen"/>
	
</launch>
